fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -318,11 +314,6 @@ protected void addFinishedQueryDetail() {
             queryDetail.setSpillBytes(statistics.spillBytes == null ? -1 : statistics.spillBytes);
         }
 
-        if (Config.enable_profile_log) {
-            String jsonString = GSON.toJson(queryDetail);
-            PROFILE_LOG.info(jsonString);
-        }
-
         QueryDetailQueue.addQueryDetail(queryDetail);
```

Comment:
Does QueryDetailQueue have a memory limit to avoid overuse memory?

Reply:
public static long query_detail_cache_time_nanosecond = 30000000000L; this config controls when to remove old queryDetail.